etc/kayobe/kolla/globals.yml

```diff
@@ -10,12 +10,20 @@ enable_docker_repo: "{% raw %}{{ 'overcloud' not in group_names or ansible_facts
 # This is necessary for os migrations where mixed clouds might be deployed
 kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}"
 
-kayobe_image_tags:
-  openstack:
-    rocky: zed-rocky-9-20230821T155947
-    ubuntu: zed-ubuntu-jammy-20230821T155947
-
-openstack_tag: "{% raw %}{{ kayobe_image_tags['openstack'][kolla_base_distro] }}{% endraw %}"
+# Dict of Kolla image tags to deploy for each service.
+# Each key is the tag variable prefix name, and the value is another dict,
+# where the key is the OS distro and the value is the tag to deploy.
+# NOTE: This is defined in etc/kayobe/kolla-image-tags.yml.
+kolla_image_tags:
+{{ kolla_image_tags | to_nice_yaml(indent=2) }}
+
+{% for tag in kolla_image_tags %}
+{% if tag == 'openstack' %}
+{{ tag }}_tag: "{% raw %}{{ kolla_image_tags['{% endraw %}{{ tag }}{% raw %}'][kolla_base_distro] }}{% endraw %}"
+{% else %}
+{{ tag }}_tag: "{% raw %}{{ kolla_image_tags['{% endraw %}{{ tag }}{% raw %}'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
+{% endif %}
+{% endfor %}
```

Comment:
Worth cherry picking to yoga?

Reply:
Possibly, just don't like to change things too much on old releases. Given the tag format is changing in zed anyway that seemed like a good point to change.